modules/alloy/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Changed disconnected components to log a warning instead of throwing an error when triggering or broadcasting events.
 
+### Removed
+- `asStructOrDie` from SpecSchema as it was unused
```

Comment:
I think we tend to treat each entry as its own sentence with Keep a Changelog

```suggestion
- Removed `asStructOrDie` from SpecSchema as it was unused
```

Reply:
I mentioned this on Friday in a call, but IMO we should actually remove this entry, as `SpecSchema` isn't public. It was basically just dead code.

Reply:
Fair point, will remove